admin/workspace-management/cvms/images.md

```diff
@@ -59,6 +59,22 @@ RUN systemctl enable docker
 
 # use systemd as the init
 RUN ln -s /lib/systemd/systemd /sbin/init
+
+# Add docker-compose
+RUN curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+RUN chmod +x /usr/local/bin/docker-compose
+
+# Add a user `coder` so that you're not developing as the `root` user
+RUN useradd coder \
+      --create-home \
+      --shell=/bin/bash \
+# Add user `coder` to the `docker` group
```

Comment:
Does this work? I thought you couldn't put comments in the middle of a command in bash

Reply:
oh, not sure. i added that just to point it out. i can remove